index.bs

```diff
@@ -2021,7 +2031,31 @@ remove optional flag from the field.
 </div>
 
 <div algorithm>
-To <dfn>serialize as a list</dfn> given |iterable|, |max depth|, |child ownership|,
+To <dfn>serialize an Array-like</dfn> given |production|, |handle id|, |value|,
```

Comment:
If `|iterable|` is passed instead of the `|value|`, serialization of `set` can be implemented via `serialize an Array-like` as well. Feel free to ignore.

Reply:
I don't think that this is possible given that we still run `set internal ids` based on the `remote value` within `serialize as Array-like`.